services/api/mw2pdf/src/classes/PdfFactory.ts

```diff
@@ -28,8 +31,9 @@ export class PdfFactory {
     return outPdf
   }
 
-  static async generateTitlePagePdf(title: string, outPdf: Pdf): Promise<void> {
+  static async generateTitlePagePdf(title: string, outPdf: Pdf, pageSize: string = 'LETTER'): Promise<void> {
```

Comment:
Just curious why it's "LETTER" here but "letter" up in `extension.json`?  (I'm sure there's a reason -- not complaining.)

Reply:
Welcome to my PAIN AND SUFFERING

This was originally not a `string` and was a PDFFormat (which meant it had to be `LETTER`)

I eventually added the type casting methods that made you giggle, and didn't change it back here.

I'm considering setting no default here, actually.

Reply:
Update: no longer sets a default in this function (which is fine -- it is never called without passing the value)